contracts/InterchainTokenFactory.sol

```diff
@@ -207,13 +204,14 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
     function deployRemoteInterchainToken(
         string calldata originalChainName,
         bytes32 salt,
-        address minter,
+        address /*minter*/,
         string memory destinationChain,
         uint256 gasValue
     ) external payable returns (bytes32 tokenId) {
         if (bytes(originalChainName).length != 0) revert NotSupported();
+        bytes memory minter_ = new bytes(0);
 
-        tokenId = deployRemoteInterchainToken(salt, minter, destinationChain, gasValue);
+        tokenId = deployRemoteInterchainToken(salt, minter_, destinationChain, gasValue);
```

Comment:
this is incosistent with the other function. I think we should not have the same function name twice in the same contract anyway, and this function does nothing for now, since the additional parameter has to be empty. If you wanna add the parameter now and support it later then maybe remove the other function completely. If you need both you should have an internal function that executes the logic and have both external functions call it to avoid the extra external call, which I think is what happens here (test this i might be wrong).